src/Mbstring/composer.json

```
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2"
+        "php": ">=7.2",
+        "ext-iconv"
```

is that legal? should this be that?
```suggestion
        "ext-iconv": "*"
```

---

Yes you are absolutely correct, I should have take more care before committing this.